internal/pkg/service/buffer/store/store.go

```diff
@@ -11,17 +12,31 @@ import (
 )
 
 type Store struct {
+	clock  clock.Clock
```

Comment:
Store vznikol ako prvy a nepouziva `clock`, potrebujem to pre jednoduchsie otestovanie.